### ISSUE TITLE
Move Catalog link below all the displayed "Dashboard" for users with multiple roles

### ISSUE THREAD

**Issue by blesildaramirez** [pkp/pkp-lib#11071]
## Issue Description
Currently, the Catalog link is displayed at the bottom of the submenu links for OMP. It should be moved after all the submission dashboards in the menu (Dashboards, Review Assignments, My Submissions):

![Image](https://github.com/user-attachments/assets/e357204e-737a-4784-b145-a6cd82e533a7)

## How to reproduce / Current behavior
Login as a user with Author, Press manager, Internal Reviewer roles. Multiple submission dashboards should appear on the sidemenu. 

## What application are you using?
 OMP 3.5.0

## Testing method
- Manual

## Action
Move the Catalog link below the last Submissions dashboard link: (e.g from the screenshot: "My Submissions")

---
Pull requests:
**OMP**
stable-3_5_0: https://github.com/pkp/omp/pull/1893
main: https://github.com/pkp/omp/pull/1894
